webpack/assets/javascripts/services/ExtendableRegistry/index.js

```diff
@@ -0,0 +1,17 @@
+const components = {};
```

Comment:
About this file:

The name is a bit confusing if we have `Slot` and `Fill` what is `ExtendableRegistry`?
What about calling this file `SlotsRegistry`?

About `add` and `remove`, we can name them as `fillSlot` and `unfillSlot`?

Reply:
I renamed the `add` and `remove` due to a prior review, I think it enough clear, plus this implementation is transparent to the consumer